charter.md

```diff
@@ -0,0 +1,51 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus once per calendar year. There's no term limit; the same individual can be reappointed as chair.
```

Comment:
nit: Putting myself as a simple person outside, I was searching on https://web-platform-tests.org/admin/index.html and the github repo and was unable to answer the following questions:


* Who is the current chair?
* When is the rotation?
* Who were the previous chairs?

Maybe it doesn't matter. not sure. Maybe it could be written on the README.

Reply:
We don't have a current chair because we don't have a charter, but @nairnandu has been acting in the role described here. I'd like to nominate @nairnandu for the chair position when the charter is accepted, but of course it's a group decision.

When decided, I think we should put the 6 participating organizations and the current chair in the top-level README.